pages/common/supervisorctl.md

```diff
@@ -0,0 +1,24 @@
+# supervisorctl
+
+> Supervisor is a client/server system that allows its users to control a number of processes on UNIX-like operating systems.
+> Supervisorctl is the command-line client piece of the supervisor which provides a shell-like interface.
+
+- Start/stop/restart a process:
+
+`supervisorctl {{start|stop|restart}} {{process name}}`
+
+- Start/stop/restart all processes in a group:
+
+`supervisorctl {{start|stop|restart}} {{group name}}:*`
+
+- Show last N **bytes** of process stderr:
+
+`supervisorctl tail -{{N}} {{process name}} stderr`
+
+- Keep showing named process stdout until ctrl-c:
```

Comment:
"named process stdout" does not seem correct. A better phrasing would be - "Continuously display the stdout of the process until ctrl-c is pressed." (I don't think the ctrl-c part is even required because a lot of things can happen other than just ctrl-c, due to which the tailing might stop)